src/Sentry.Extensions.Logging/BindableSentryLoggingOptions.cs

```diff
@@ -9,11 +9,22 @@ internal class BindableSentryLoggingOptions : BindableSentryOptions
     public LogLevel? MinimumEventLevel { get; set; }
     public bool? InitializeSdk { get; set; }
 
+    [Experimental(Infrastructure.DiagnosticId.ExperimentalFeature)]
```

Comment:
Not sure we need these (it's an `internal` class)... just more work to clean them up afterwards.

Reply:
indeed ...
removed all `ExperimentalAttribute` from `internal` types and members